example/pg.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
I don't think we need to create this [singleton pattern](https://en.wikipedia.org/wiki/Singleton_pattern) for the pg pool. We can keep it simple by creating the pool in the init function, so the pool will only be created once when you initialise your server with init